python-src/resources.py

```diff
@@ -39,7 +39,8 @@ def post(self):
         game = Game()
         request_data = request.get_json()
         game.players = [ndb.Key(urlsafe=request_data['player_id'])]
-        game.cities = Util.get_cities(int(request_data.get('number_of_cities', 3)))
+        game.cities = Util.get_cities(int(request_data.get('number_of_cities', 3)),
```

Comment:
How to use this difficulty in frontend? I didn't find a place to declare the difficulty of cities when POST a game. Is it possible to add it in route when creating and searching for games? or there is some other way?